README.md

```diff
@@ -43,6 +43,8 @@ Current stable release is able to persist Orion context data in:
 * [MongoDB](https://www.mongodb.org/), the NoSQL document-oriented database.
 * [STH](https://github.com/telefonicaid/IoT-STH), a Short-Term Historic database built on top of MongoDB.
 
+You may consider to visit [Cygnus Quick Start Guide](doc/quick_start_guide.md) before going deep into the details.
```

Comment:
Both may be correct, but the second one is definitely more elegant.
However ... :-)
We have too many errors of this kind to really worry, right? ;-)

NTC